python/ray/util/state/common.py

```diff
@@ -518,6 +518,8 @@ class NodeState(StateSchema):
     end_time_ms: Optional[int] = state_column(
         filterable=False, detail=True, format_fn=Humanify.timestamp
     )
+    #: Cloud instance id if provided when starting the node.
+    instance_id: Optional[str] = state_column(filterable=True, detail=True)
```

Comment:
I don't think we need this from the state API. Maybe it is more useful to have a instance type rather than instance id?

Reply:
yeah, i added it mainly so i could test it e2e. otherwise, it's pretty hard to test.

Reply:
maybe just use the state_manager's client directly?